modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -3,28 +3,28 @@
     class="apos-table__cell-field apos-table__cell-field--labels"
     :class="`apos-table__cell-field--${header.name}`"
   >
-    <span v-if="manuallyPublished && item.modified && !item.submitted && item.lastPublishedAt">
+    <span v-if="manuallyPublished && draft.modified && draft.submitted && draft.lastPublishedAt">
       <AposLabel
         label="Pending Updates" class="apos-table__cell-field__label"
         tooltip="There are active changes to this document."
         :modifiers="[ 'is-success', 'is-filled' ]"
       />
     </span>
-    <span v-else-if="item.submitted">
+    <span v-else-if="draft.submitted">
       <AposLabel
         label="Pending" class="apos-table__cell-field__label"
         tooltip="Changes to this document are awaiting approval by an admin or editor."
         :modifiers="[ 'is-filled' ]"
       />
     </span>
-    <span v-if="manuallyPublished && !item.lastPublishedAt">
+    <span v-if="manuallyPublished && !draft.lastPublishedAt">
```

Comment:
Seems like an unnecessary change.

Reply:
We do eventually copy this to `published`, but I believe it is best to get anything relating to publication status from the draft, which is the primary place Apostrophe stores it.